Fix broken links in the readme. (#342)
plone/plonetheme.barceloneta: README.md

Latest version on [npmjs.com](https://www.npmjs.com/package/@plone/plonetheme-barceloneta-base)

## Documentation

> Plonetheme.barceloneta documentation https://6.docs.plone.org/classic-ui/theming/barceloneta.html is part of the official Plone documentation.

Reviewer: ```suggestion
The documentation for `plonetheme.barceloneta` is part of the official Plone documentation, [Classic UI theming based on Barceloneta](https://6.docs.plone.org/classic-ui/theming/barceloneta.html).
```

## Changelog
https://github.com/plone/plonetheme.barceloneta/blob/master/CHANGES.md

## Source Code and Contribution

### Contributors

> Please read the document [Process for Plone core's development](https://6.docs.plone.org/contributing/index.html). Also consult the section about contribution in the [plone.api contributors documentation](https://6.docs.plone.org/plone.api/contribute/index.html).

Reviewer: ```suggestion
Please read the document [Contributing to Plone](https://6.docs.plone.org/contributing/index.html). Also consult the document [Contributing to `plone.api`](https://6.docs.plone.org/plone.api/contribute/index.html).
```
